gradle.properties

```diff
@@ -0,0 +1,2 @@
+// For the gRPC client sample.
+android.useAndroidX=true
```

Comment:
oooooooh is this the fix I’ve been needing?!

Reply:
Don't think so?